### PR TITLE
Update Developer Guide

### PR DESCRIPTION
**Pull request by pjanthony2001**
Initial commit where small changes were made to the product scope

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -325,7 +368,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 1a1. AddressBook shows an error message.
+
+      Use case ends.
+
+**Use case: Clear all clients**
+
+**MSS**
+
+1.  User requests to clear all clients
+2.  ConnectCare clears all clients
```
Comment (Jolonauh) [inline docs/DeveloperGuide.md:393]: Should the system confirm with the user first before clearing all clients?

**Review by Jolonauh — COMMENTED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (pjanthony2001) [reply docs/DeveloperGuide.md:393]: Thats probably a good idea

**Comment by Jolonauh** [comment]
LGTM

**Pull request merged**